ios/FluentUI/Badge Field/BadgeView.swift

```diff
@@ -297,16 +326,24 @@ open class BadgeView: UIView {
             accessibilityHint = "Accessibility.Select.Hint".localized
         }
     }
+    
+    private func updateColors() {
+        updateBackgroundColor()
+        updateLabelTextColor()
+    }
 
     private func updateBackgroundColor() {
         if let window = window {
-            backgroundView.backgroundColor = backgroundColor(for: window, style: style, selected: isSelected, enabled: isActive)
+            let customColor = isSelected ? backgroundSelectedColor : backgroundColor
```

Comment:
So, if the badge is disabled and the background custom colors are set, are we getting the gray look?

Reply:
no, my understanding is that disabled state is only supported for default style and not others. If the badge is initialized in default mode it will respect the gray look not otherwise.

Reply:
that seems broken. client should be able to disable the badge view and then when it re-enables it will show the custom colors again.

Reply:
I have updated changes to reflect this. setting ` isActive = false ` will now restore the original custom colors. 
Setting it to `true` value will set it to custom (if set one) disabled appearance.